tests/test_configure.py

```diff
@@ -454,6 +454,13 @@ def test_read_config_from_environment_variables() -> None:
         ):
             fresh_pydantic_plugin()
 
+    with patch.dict(os.environ, {'LOGFIRE_SEND_TO_LOGFIRE': 'not-valid'}):
+        with pytest.raises(
```

Comment:
This is a good place to use https://15r10nk.github.io/inline-snapshot/extra/#inline_snapshot.extra.raises, mostly just to avoid escaping the regex.

Reply:
Cool done!